TTS/utils/audio/torch_transforms.py

```diff
@@ -0,0 +1,165 @@
+import librosa
+import torch
+from torch import nn
+
+
```

Comment:
I don't know how useful it would be, but maybe it would be nice to have a differentiable equivalent of functions in numpy_transform here when possible.